src/pkg/build/conveyorPacker_debootstrap.go

```diff
@@ -97,32 +97,32 @@ func (cp *DebootstrapConveyorPacker) Pack() (b *Bundle, err error) {
 	//move downloaded files from tmpdir to bundle
 	err = os.Rename(cp.tmpfs, b.Rootfs())
 	if err != nil {
-		return nil, fmt.Errorf("Failed to move rootfs into bundles rootfs: %v", err)
+		return nil, fmt.Errorf("While renaming bundle rootfs: %v", err)
 	}
 
 	//change root directory permissions to 0755
 	if err := os.Chmod(b.Rootfs(), 0755); err != nil {
-		return nil, fmt.Errorf("Failed to change file permissions of rootfs: %v", err)
+		return nil, fmt.Errorf("While changing bundle rootfs perms: %v", err)
 	}
 
 	err = cp.insertBaseEnv(b)
 	if err != nil {
-		return nil, fmt.Errorf("Failed to insert base env: %v", err)
+		return nil, fmt.Errorf("While inserting base environtment: %v", err)
 	}
 
 	err = cp.insertRunScript(b)
 	if err != nil {
-		return nil, fmt.Errorf("Failed to insert default runscript: %v", err)
+		return nil, fmt.Errorf("While inserting runscript: %v", err)
 	}
 
 	return b, nil
 }
 
 func (cp *DebootstrapConveyorPacker) insertBaseEnv(b *Bundle) (err error) {
 	if err = makeBaseEnv(b.Rootfs()); err != nil {
-		sylog.Errorf("%v", err)
+		return
 	}
-	return
+	return nil
```

Comment:
This was fine with "return", since no error occurred err should still be nil going out.